docs/schema_mapping.py

```diff
@@ -107,6 +107,20 @@
         'JSON Schema Validation',
         'All the literal values in the enum are included in the definition.'
     ],
+    [
+        'SecretStr',
+        'string',
+        '{"writeOnly": true}',
+        'Pydantic standard "format" extension',
+        ''
+    ],
+    [
+        'SecretBytes',
+        'string',
+        '{"writeOnly": true}',
+        'Pydantic standard "format" extension',
```

Comment:
Sorry, forgot to mention. After removing `format`, this no longer applies: `'Pydantic standard "format" extension',`.